diff --git a/Resources/Private/PHP/Composer/respect/rest/library/Respect/Rest/Routes/AbstractRoute.php b/Resources/Private/PHP/Composer/respect/rest/library/Respect/Rest/Routes/AbstractRoute.php
index 9918bee..0dc999c 100644
--- a/library/Respect/Rest/Routes/AbstractRoute.php
+++ b/library/Respect/Rest/Routes/AbstractRoute.php
@@ -26,7 +26,7 @@ use Respect\Rest\Routines\Unique;
  * @method \Respect\Rest\Routes\AbstractRoute acceptLanguage()
  * @method \Respect\Rest\Routes\AbstractRoute accept()
  * @method \Respect\Rest\Routes\AbstractRoute authBasic()
- * @method \Respect\Rest\Routes\AbstractRoute by()
+ * @method \Respect\Rest\Routes\AbstractRoute by($callback)
  * @method \Respect\Rest\Routes\AbstractRoute contentType()
  * @method \Respect\Rest\Routes\AbstractRoute ignorableFileExtension()
  * @method \Respect\Rest\Routes\AbstractRoute lastModified()
